docs/source/topics/known-issues.rst

```diff
@@ -102,3 +102,12 @@ Caching
 Secondly, you must use the :class:`django_mongodb_backend.cache.MongoDBCache`
 backend rather than Django's built-in database cache backend,
 ``django.core.cache.backends.db.DatabaseCache``.
+
+Queryable Encryption
+====================
+
+Consider these
+`limitations and restrictions <https://www.mongodb.com/docs/manual/core/queryable-encryption/reference/limitations/>`_
+before enabling Queryable Encryption. Some operations are unsupported, and others behave differently.
```

Comment:
I was hoping for a translation of how those restrictions apply to Django querysets, etc. Quickly scanning that page, it reads as MongoDB mumbo jumbo that's meaningless to Djangonauts.

Reply:
The [tutorial](https://github.com/mongodb/docs/blob/adad2b1ae41ec81a6e5682842850030813adc1e5/source/includes/qe-tutorials/python/queryable_encryption_tutorial.py) only demonstrates `find_one` and I've been using `get` in the tests. I'm not sure if `filter` and/or lookups are going to work.